frontend/src/views/web/challenge/submission.html

```diff
@@ -7,8 +7,9 @@
                         <strong>Submission Guidelines</strong>
                     </span>
                     <div ng-if="challenge.isChallengeHost">
-                        <strong class="red-text">
-                            <p>Your submissions won't be visible on leaderboard since you are a Challenge Host. This feature is only for your testing purposes.</p>
+                        <strong>
+                            <p>Your submissions won't be visible on leaderboard for <strong class="red-text"> Public Challenge Phases </strong> since you are a Challenge Host.<br>
```

Comment:
Please change the text to 

```
<p><strong>Note: </strong>Your submissions won't be visible on leaderboard for <strong class="blue-text"> Public Challenge Phases </strong> since you are a challenge host for this challenge.<br> This feature is only for your testing purposes.</p>
```